tests/telemetry/test_writer.py

```diff
@@ -242,6 +242,56 @@ def test_add_integration(telemetry_lifecycle_writer, test_agent_session, mock_ti
     assert requests[0]["body"] == _get_request_body(expected_payload, "app-integrations-change")
 
 
+# New test for app_configurations_changed_event
```

Comment:
Neither of these comments add information to what's communicated by the lines below them. I recommend removing them.